controllers/admin/AdminTabsController.php

```diff
@@ -31,17 +31,27 @@ class AdminTabsControllerCore extends AdminController
 {
     protected $position_identifier = 'id_tab';
 
+    /**
+     * @deprecated since 1.7.8, to be removed in the next minor
```

Comment:
Please deprecate the class : https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations